package.json

```diff
@@ -68,5 +79,9 @@
     "url": "https://github.com/kenwheeler/nuka-carousel/issues"
```

Comment:
Unrelated, but can you update the `https://github.com/kenwheeler/` strings to `https://github.com/FormidableLabs/`? (missed from when Ken ported over the repo).